doc/source/index.rst

```diff
@@ -240,13 +242,33 @@ which necesitate more resources.
   .. image:: technology_showcase_examples/images/thumb/sphx_glr_tse-001-brake_squeal_thumb.png
     :alt: Brake Squeal Analysis
 
-  :ref:`sphx_glr_ex_0-tecbrakesqueal.py`
+  :ref:`sphx_glr_ex_0-tecbrakesqueal.rst`
 
 .. raw:: html
 
       <div class="sphx-glr-thumbnail-title">Brake Squeal Analysis</div>
     </div>
 
+.. ##########
+
+.. raw:: html
+
+    <div class="sphx-glr-thumbcontainer" tooltip="Technology Showcase Example 15: Calibrating and Validating a Hyperelastic Constitutive Model">
+
+.. only:: html
+
+  .. image:: technology_showcase_examples/images/thumb/sphx_glr_tse-015-calvalhyper_thumb.gif
+    :alt: Calibrating and Validating a Hyperelastic Constitutive Model
+  
+  :ref:`sphx_glr_ex_15-teccalvalhyper.rst`
+
+.. raw:: html
+
+      <div class="sphx-glr-thumbnail-title">Calibrating and Validating a Hyperelastic Constitutive Model</div>
+    </div>
+
+.. #########
```

Comment:
I guess this is generated by sphinx?

Reply:
I extended it: I don't have an example repository so I don't generate the Sphinx gallery like PyMAPDL or PyMAPDL-examples do. Let me know if it's not good practice!